wp-content/civi-extensions/goonjcustom/Civi/CollectionCampOutcomeService.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace Civi;
+
+use Civi\Api4\Contact;
+use Civi\Api4\EckEntity;
+use Civi\Core\Service\AutoSubscriber;
+
+/**
+ *
+ */
+class CollectionCampOutcomeService extends AutoSubscriber {
+
+  /**
+   * Define subscribed events.
+   */
+  public static function getSubscribedEvents() {
+    return [];
+  }
+
+  /**
+   * Process camp reminder logic.
+   *
+   * @param array $camp
+   * @param \DateTimeImmutable $today
+   *
+   * @throws \Exception
+   */
+  public static function processCampReminder($camp, $today) {
+    $from = self::getDefaultFromEmail();
```

Comment:
This can be passed from the caller function – in our case from the cron. And here it will be a param